addons/website_event/models/event_event.py

```diff
@@ -507,6 +507,14 @@ def _track_subtype(self, init_values):
             return self.env.ref('website_event.mt_event_unpublished', raise_if_not_found=False)
         return super(Event, self)._track_subtype(init_values)
 
+    def _get_external_description(self):
+        # Adding the URL of the event into the description
+        self.ensure_one()
+        max_length = 1900
```

Comment:
As said before, would say either put the initial max_length at 1900 or keep 2000 + the URL here.

If each module starts re-defining a new max_length, it becomes a bit hard to follow.
Will also simplify the code, as this just needs to append the URL and be done.